plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java

```diff
@@ -94,7 +94,7 @@ private BigQueryType() {}
             1, // 9 digits after the dot
     };
     private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("''HH:mm:ss.SSSSSS''");
-    private static final DateTimeFormatter DATETIME_FORMATTER = DateTimeFormatter.ofPattern("''yyyy-MM-dd HH:mm:ss.SSSSSS''");
+    private static final DateTimeFormatter DATETIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
```

Comment:
`yyyy` leads to incorrect result. Please reuse `BigQueryTypeUtils#DATETIME_FORMATTER`

Reply:
I think for now I'll just fix that here and then do a follow up PR were we rely only on `BigQueryType#DATETIME_FORMATTER` and also update https://github.com/trinodb/trino/blob/f1f45bb38281c1565194528844880841132aa68d/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeUtils.java#L103-L108